src/fcs.jl

```diff
@@ -8,12 +8,17 @@ const ETHERTYPE_IPV4 = UInt16(0x0800)
 const IP_TOTAL_LENGTH_OFFSET = ETHERNET_HEADER_SIZE + 2
 
 """
-    pcap_has_fcs(::PcapReader)
+    pcap_has_fcs(::PcapReader; confirm_checksum = true) -> Union{Nothing, Bool}
 
 Heuristically determine whether captured frames contain Ethernet FCS or not.
 Unfortunately, PCAP format doesn't provide this information explicitly.
+
+If it is not possible to determine the status, return `nothing`.
```

Comment:
This would not be obvious from the function name. If it fails it would better if it throws, and if it has a ternary result it should be clear in the name, e.g., `try_detect_fcs`.

Reply:
Is there a good existing error type to throw? I would be slightly more inclined to use the `try_` name to emphasise the heuristic nature of the function.

Reply:
I wouldn't make it too complicated, I just want it to be clear. When this was put in I argued if it says "has" it should return true only if it's detected presence, otherwise false. If we want a function that returns true/false/undetermined that's a new function. We could have for example
```Julia
fcs_presence() -> :none | :present | :undetermined
has_fcs() = fcs_presence() == :present
```

Reply:
Done: I've split the functions and tests into `try_detect_fcs` and `pcap_has_fcs`